src/Compilers/VisualBasic/Portable/Syntax/Syntax.xml

```diff
@@ -5084,25 +5084,43 @@ Unify Enum declarations with other block type declarations.
     <!--********************
       -  TupleElementSyntax
       **********************-->
-    <node-structure name="TupleElementSyntax" parent="VisualBasicSyntaxNode">
+    <node-structure name="TupleElementSyntax" abstract="true" parent="VisualBasicSyntaxNode">
       <description>Represents a single declaration of a tuple element. </description>
       <grammar>TupleElement</grammar>
+    </node-structure>
 
-      <node-kind name="TupleElement"></node-kind>
+    <!--********************
+      -  TypedTupleElementSyntax
+      **********************-->
+    <node-structure name="TypedTupleElementSyntax" parent="TupleElementSyntax">
```

Comment:
Actually neither seem good names to be certainly final. 
I will keep the current and enter an issue to follow up. Need to figure the names by RC though.

---
In reply to: [82266726](https://github.com/dotnet/roslyn/pull/14284#discussion_r82266726) [](ancestors = 82266726,82260796)